test/fuzzer.cpp

```diff
@@ -49,6 +53,33 @@ int main()
 		input += s + '\n';
 	}
 
+	if (standard)
+	{
+		string outputString(compileStandard(input.c_str(), NULL));
+		Json::Value output;
+		if (!Json::Reader().parse(outputString, output))
+		{
+			cout << "Compiler produced invalid JSON output." << endl;
+			abort();
+		}
+		if (output.isMember("errors"))
+			for (auto const& error: output["errors"])
+			{
+				string invalid = contains(error["type"].asString(), vector<string>{
+					"JSONError",
```

Comment:
We should not flag an error on JSONError, otherwise the fuzzer will only "find" invalidly encoded input, I think.